app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.jetpack.compose.learning">
 
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```

Comment:
Why we need this in manifest?

Reply:
It was showing error with <uses-permission block so I replaced it with <uses-feature as recommended

Reply:
Did you mention this thing in the PR description?

Reply:
Done.
Also, for every warnings etc small things do I need to mention it particularly in description? Isn't something like this understandable?

Reply:
No, but at least we can mention if things are irrelevant for the PR.